src/lib.rs

```diff
@@ -62,6 +67,20 @@ pub struct InnerCommand {
     env: HashMap<String, String>,
 }
 
+impl InnerCommand {
+    fn to_stdlib_command(&self) -> process::Command {
```

Comment:
Looking at where you used this method, you could safely consume `self` here and save yourself a bunch of cloning.

Reply:
sure